src/eval-frame/tools/send-file-request-to-editor.js

```diff
@@ -0,0 +1,83 @@
+import generateRandomId from "../../shared/utils/generate-random-id";
+import messagePasserEval from "../../shared/utils/redux-to-port-message-passer";
+
+// NOTE: exporting this for tests.
+export const fileRequestQueue = {};
+
+export function onParentContextFileRequestSuccess(
```

Comment:
this function and the one below it are very close in spirit to `onParentContextFileFetchSuccess` etc., which was used to resolve the local file lines in fetch chunks.